src/main/java/org/onebusaway/alexa/AuthedSpeechlet.java

```diff
@@ -35,17 +34,13 @@
     @Resource
     private AnonSpeechlet anonSpeechlet;
 
-    @Resource
-    private ObaAgencies obaAgencies;
-
     private ObaUserClient obaUserClient;
 
     private ObaUserDataItem userData;
 
     public void setUserData(ObaUserDataItem userData) {
         this.userData = userData;
-        this.obaUserClient = new ObaUserClient(
-                obaAgencies.agencyForCity(userData.getCity()).get());
+        this.obaUserClient = new ObaUserClient(userData.getObaBaseUrl());
```

Comment:
In combination with my other comments -- here's where we'll need to decide how to handle `userData.getObaBaseUrl()` returning null, before instantiating `ObaUserClient`.